api/HTMLSourceElement.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/embedded-content.html#htmlsourceelement",
         "support": {
           "chrome": {
-            "version_added": "1"
+            "version_added": "3"
```

Comment:
html.element.source just says true, but this change is consistent with audio and video elements.